src/components/Breadcrumb/Breadcrumb.scss

```diff
@@ -3,9 +3,31 @@
 
```

Comment:
@dubunin Basically you shouldn't have  @import '~office-ui-fabric/src/components/Breadcrumb/Breadcrumb';. 

I am frankly surprised there isn't a merge conflict, can you check what is in master?

Also tagging @dzearing